src/dataset-builder/ConceptSearch.ts

```diff
@@ -0,0 +1,150 @@
+            width: '100%',
+            maxWidth: 575,
+            height: '3rem',
+            marginRight: 20,
+            paddingLeft: 40,
+          },
+        }),
+        icon('search', { size: 18, style: { position: 'absolute', left: 15, top: '50%', marginTop: -9 } }),
+      ]),
+      concepts.status === 'Ready'
+        ? h(SimpleTable, {
+            'aria-label': 'concept search results',
+            underRowKey: 'underRow',
+            rowStyle: {
+              backgroundColor: 'white',
+              ...tableLeftPadding,
+            },
+            headerRowStyle: {
+              ...tableHeaderStyle,
+              ...tableLeftPadding,
+              marginTop: '1rem',
+            },
+            cellStyle: {
+              paddingTop: 10,
+              paddingBottom: 10,
+            },
+            columns: [
+              { header: strong(['Concept name']), width: 710, key: 'name' },
+              { header: strong(['Concept ID']), width: 195, key: 'id' },
+              { header: strong(['Code']), width: 195, key: 'code' },
+              { header: strong(['Roll-up count']), width: 205, key: 'count' },
+              { width: 100, key: 'hierarchy' },
+            ],
+            rows: _.map((concept) => {
+              const [label, iconName]: [string, IconId] = (() => {
+                if (_.contains(concept, cart)) {
+                  return ['remove', 'minus-circle-red'];
+                }
+                return ['add', 'plus-circle-filled'];
+              })();
+              return {
+                name: div({ style: { display: 'flex' } }, [
+                  h(Link, { 'aria-label': `${label} ${concept.id}`, onClick: () => setCart(_.xor(cart, [concept])) }, [
+                    icon(iconName, { size: 16 }),
+                  ]),
+                  div({ style: { marginLeft: 5 } }, [
+                    h(HighlightConceptName, { conceptName: concept.name, searchFilter: search }),
+                  ]),
+                ]),
+                id: concept.id,
+                count: concept.count,
+                hierarchy: div({ style: { display: 'flex' } }, [
+                  h(
+                    Link,
+                    {
+                      'aria-label': `open hierarchy ${concept.id}`,
+                      onClick: () =>
+                        onOpenHierarchy(
+                          { id: concept.id, category: domainOption.category, root: concept },
+                          cart,
+                          search
+                        ),
+                    },
+                    [icon('view-list')]
+                  ),
+                  div({ style: { marginLeft: 5 } }, ['Hierarchy']),
+                ]),
+              };
+            }, concepts.state.result),
+          })
+        : spinnerOverlay,
+    ]),
+    cart.length !== 0 &&
+      h(ActionBar, {
+        prompt: cart.length === 1 ? '1 concept selected' : `${cart.length} concepts selected`,
```

Comment:
Very optional/thinking out loud: Did you think about something like this? I am not even sure its worth it, but figured I'd suggest it:
```suggestion
        prompt: `${cart.length} concept${cart.length === 1 ? '' : 's'} selected`,
```

Reply:
My old L10N habits make me dislike building plurals like this. I figure it's about the same amount of code either way so I'd rather leave it as-is unless there's a strong convention against doing it this way.